packages/web-components/fast-foundation/src/switch/switch.template.ts

```diff
@@ -9,7 +9,6 @@ export const SwitchTemplate = html<Switch>`
     <template
         role="switch"
         aria-checked="${x => x.checked}"
-        aria-required="${x => x.required}"
```

Comment:
hmmm - I'd expect that this component shouldn't support a `required` prop / attribute either then. It inherits that property from `FormAssociated` though - I'll write up a quick issue to track